### PR TITLE
W-14030997 sap 5.9

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by valkyrie69 — DISMISSED** [review]

File: sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc [hunk 1]
```diff
@@ -312,6 +314,49 @@ tRFC can be serialized using inbound and outbound queues, hence the name queued
 * SAP:SYSTEM_BUSY
 * SAP:TIMEOUT
 
+[[asynchronousRemoteFunctionCall]]
+=== Asynchronous RFC
```
Comment (elcoya) [inline sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc:320]: I'm concerned about the naming. Why `async-rfc` translates to `Asynchronous Remote Function Call` and `asynchronousRemoteFunctionCall` into `Asynchronous RFC`? Shouldn't it be the other way around?

File: sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc [hunk 2]
```diff
@@ -262,15 +262,17 @@ SAP 5.6 and later supports updating a specific row in a table when a response is
...
 * <<getFunction>>
 * <<retrieveIdoc>>
 * <<send>>
 * <<startsaptransaction>>
 * <<syncRfc>>
+* <<synchronousRemoteFunctionCall>>
 
 [[asyncRfc]]
-=== Asynchronous Remote Function Call
+=== Asynchronous Remote Function Call (DEPRECATED)
```
Comment (elcoya) [inline sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc:275]: When deprecating, we should point to the new way of doing the same. Can we refer to the new operation `asynchronousRemoteFunctionCall`?

File: sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc [hunk 3]
```diff
@@ -529,7 +574,7 @@ Creates a transaction ID to use as part of future calls.
 
 
 [[syncRfc]]
-=== Synchronous Remote Function Call
+=== Synchronous Remote Function Call (DEPRECATED)
```
Comment (elcoya) [inline sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc:579]: Please link the replacing with `synchronousRemoteFunctionCall` somehow

**Review by elcoya — CHANGES_REQUESTED** [review]
@sathya0 Can you review my inline comments please

File: sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc (hunk 1 above)
Comment (sathya0) [reply sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc:320]: This is accurate as verified on the Slack channel, not sure why though

**Review by elcoya — APPROVED** [review]
Looks good

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**